### DISCUSSION TITLE
How can I use the GUI with other models and CPU-only mode?

### DISCUSSION THREAD

**Discussion by cyatarow** [azuritecoin/OnnxDiffusersUI#9] [Q&A] [answered]
I cannot find a model selection field or a toggle of CPU-only mode in the GUI.
How do I use them?

**Comment by azuritecoin** [comment, marked as answer]
First make sure you updated to the latest duffusers v0.6.0 and got the latest `onnxUI.py` script downloaded.

The way you select the model is with the `--model` flag when starting the UI script. For example to run the base SD model you would do:  
```bash
python onnxUI.py --model="./stable_diffusion_onnx"
```

To run CPU only mode, add a `--cpu-only' flag.  
```bash
python onnxUI.py --model="./stable_diffusion_onnx" --cpu-only
```

**Reply by cyatarow** [reply]
Thank you! They work fine.
